matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -260,3 +326,32 @@ async fn run_socket(
         }
     }
 }
+
+pub(crate) fn new_senders_and_receivers<T>(
+    config: &WebRtcSocketConfig,
+) -> (Vec<UnboundedSender<T>>, Vec<UnboundedReceiver<T>>) {
+    (0..config.channels.len())
+        .map(|_| futures_channel::mpsc::unbounded())
+        .unzip()
+}
+
+fn channel_readiness(
```

Comment:
It's hard to guess from the name what the function is about...

Perhaps `create_channels_ready_fut"?

Reply:
Or maybe rather `create_data_channels_ready_fut` there are so many channels here, mpsc channels for instance, probably better to err on the verbose side.

Reply:
Happy to change it